.circleci/config.yml

```diff
@@ -21,43 +24,61 @@ workflows:
               only: /^v.+\..+\..+/
   tests:
     jobs:
-      - test_installation
+      - test-installation
 
 
 jobs:
   lint:
     docker:
-      - image: circleci/python:3.6
+      - image: cimg/python:3.6
     steps:
       - checkout
-      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-          | python3 - --version << pipeline.parameters.poetry_version >>"
+      - install-poetry
       - run: poetry install --no-root
       - run: poetry run isort --check .
       - run: poetry run black --check .
       - run: poetry run mypy -p "brainframe.cli"
-  upload_to_pypi:
+
+  build-deb:
+    docker:
+      - image: cimg/base:stable-18.04
```

Comment:
I'm not sure. It doesn't seem like the resulting deb has any attachment to 18.04 specifically, and the deb installs fine on my local 20.04 installation. I imagine this will become more clear when we start trying to put this thing in a package repository.

Reply:
Turns out you were right. I'm not sure how it worked before, but building the deb on one version and installing it on another breaks some stuff. I've made the `build-deb` step a matrix build for 18.04 and 20.04.